.github/workflows/main.yaml

```diff
@@ -95,11 +100,13 @@ jobs:
           go-version-file: go.mod
           cache-dependency-path: go.sum
 
+      - name: Setup kubectl
+        uses: marcofranssen/setup-kubectl@v1.3.0
+        with:
+          kubectlVersion: ${{ env.KUBECTL_VERSION }}
+
       - name: Install dependences
-        run: |
-          command -v ginkgo || go install github.com/onsi/ginkgo/v2/ginkgo@latest
-          go install sigs.k8s.io/kind@v0.17.0
-          curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.25.1/bin/linux/amd64/kubectl && sudo install kubectl /usr/local/bin/kubectl
+        run: make ginkgo kind
```

Comment:
See how the makefile is now controlling these tool versions and doing a `go install` instead of `go get`.